api/app.js

```diff
@@ -109,60 +107,67 @@ let verifySession = (req, res, next) => {
  * Purpose: Get all lists that belong to an authenticated user
  * Return: Array of all lists
  */
-app.get('/lists', authenticate, (req, res) => {
+app.get('/lists', authenticate, async (req, res) => {
     // We want to return an array of all the lists that belong to the authenticated user 
-    List.find({
-        _userId: req.user_id
-    }).then((lists) => {
-        res.send(lists);
-    }).catch((err) => {
+    try {
+        res.send(await List.find({
+            "_userId": req.user_id
+        }))
+    } catch (err) {
         res.send(err);
-    });
+    }
 })
 
 /**
  * POST /lists
  * Purpose: Create new list
  * Return: New list document 
  */
-app.post('/lists', authenticate, (req, res) => {
-    let title = req.body.title;
+app.post('/lists', authenticate, async (req, res) => {
     let newList = new List({
-        title,
-        _userId: req.user_id
-    });
-    newList.save().then((listDoc) => {
-        //full list document is returned
-        res.send(listDoc);
+        "title": req.body.title,
```

Comment:
Wasn't really a need to initialize a variable, and i quoted the props cause of eslint